src/pages/Mypage/Mypage.js

```diff
@@ -1,22 +1,49 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import MypageTop from './components/MypageTop';
 import MenuBar from './components/MenuBar';
 import OrderDelivery from './components/OrderDelivery';
 import Subscribes from './components/Subscribes';
 import Likes from './components/Likes';
+import ConfirmPassword from './components/ConfirmPassword';
 import './Mypage.scss';
 
+const token =
+  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaWF0IjoxNjgzNjM4NDA0fQ.9AnFo7VZuBaLGv9jSTIq3XFd5PBZOKQpUchEfWzAT80';
+
```

Comment:
```suggestion
const token = localStorage.getItem("token")
```

물리적으로 토큰 값 입력하는 방법 대신 브라우저 스토리지에 담긴 토큰을 꺼내서 사용하는 방식으로 구현해주세요!